pkgs/development/libraries/aribb25/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitLab
+, fetchpatch
+, autoreconfHook
+, pkgconfig
+, pcsclite
+, PCSC
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aribb25";
+  version = "0.2.7";
+
+  src = fetchFromGitLab {
+    domain = "code.videolan.org";
+    owner = "videolan";
+    repo = pname;
+    # rev = version; FIXME: uncomment in next release
```

Comment:
Maybe move this comment to the `version` attribute instead, where it's more likely to be seen when bumping the version. It will also show up in the GitHub diff in that case.

Reply:
Fixed.